flake8_eradicate.py

```diff
@@ -121,19 +125,28 @@ def _contains_commented_out_code(self) -> bool:
         To prevent this false-positive, the tokens of the physical line are
         checked for a comment. The eradicate function is only invokes,
         when the tokens indicate a comment in the physical line.
-
         """
-        comment_in_line = any(
-            token_type == tokenize.COMMENT
-            for token_type, _, _, _, _ in self._tokens
-        )
-
-        if comment_in_line:
-            filtered_source = ''.join(
-                self._eradicator.filter_commented_out_code(
-                    self._physical_line,
-                    self._options['aggressive'],
-                ),
+        with open(self.filename) as f:
```

Comment:
So, my main question right now is: why do we have to construct our tokens here, why can't we use `file_tokens` object from `flake8`?

Reply:
I changed the code to make it use file_tokens in the constructor, I'm not sure it's improving a lot since we still have to open the file to read the physical lines but if that looks better to you, so be it